java container builds are failing with the below error

```
 0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (6) Could not resolve host: sbt-downloads.cdnedge.bluemix.net
The command '/bin/sh -c curl -LO "https://services.gradle.org/distributions/gradle-${GRADLE_VERSION}-bin.zip"     && unzip -q gradle-${GRADLE_VERSION}-bin.zip -d /opt/     && chmod +x /opt/gradle-${GRADLE_VERSION}/bin/gradle     && rm gradle-${GRADLE_VERSION}-bin.zip     && curl -LO "https://downloads.apache.org/maven/maven-3/${MAVEN_VERSION}/binaries/apache-maven-${MAVEN_VERSION}-bin.zip"     && unzip -q apache-maven-${MAVEN_VERSION}-bin.zip -d /opt/     && chmod +x /opt/apache-maven-${MAVEN_VERSION}/bin/mvn     && rm apache-maven-${MAVEN_VERSION}-bin.zip     && curl -LO "https://sbt-downloads.cdnedge.bluemix.net/releases/v${SBT_VERSION}/sbt-${SBT_VERSION}.zip"     && unzip -q sbt-${SBT_VERSION}.zip -d /opt/     && chmod +x /opt/sbt/bin/sbt     && rm sbt-${SBT_VERSION}.zip     && curl -LO "https://github.com/koalaman/shellcheck/releases/download/v${SHELLCHECK_VERSION}/shellcheck-v${SHELLCHECK_VERSION}.linux.x86_64.tar.xz"     && tar -C /tmp/ -xvf shellcheck-v${SHELLCHECK_VERSION}.linux.x86_64.tar.xz     && cp /tmp/shellcheck-v${SHELLCHECK_VERSION}/shellcheck /usr/local/bin/shiftleft/shellcheck     && chmod +x /usr/local/bin/shiftleft/shellcheck     && curl -LO "https://github.com/dominikh/go-tools/releases/download/${SC_VERSION}/staticcheck_linux_amd64.tar.gz"     && tar -C /tmp -xvf staticcheck_linux_amd64.tar.gz     && chmod +x /tmp/staticcheck/staticcheck     && cp /tmp/staticcheck/staticcheck /usr/local/bin/shiftleft/staticcheck     && rm staticcheck_linux_amd64.tar.gz' returned a non-zero code: 6
##[error]The command '/bin/sh -c curl -LO "https://services.gradle.org/distributions/gradle-${GRADLE_VERSION}-bin.zip"     && unzip -q gradle-${GRADLE_VERSION}-bin.zip -d /opt/     && chmod +x /opt/gradle-${GRADLE_VERSION}/bin/gradle     && rm gradle-${GRADLE_VERSION}-bin.zip     && curl -LO "https://downloads.apache.org/maven/maven-3/${MAVEN_VERSION}/binaries/apache-maven-${MAVEN_VERSION}-bin.zip"     && unzip -q apache-maven-${MAVEN_VERSION}-bin.zip -d /opt/     && chmod +x /opt/apache-maven-${MAVEN_VERSION}/bin/mvn     && rm apache-maven-${MAVEN_VERSION}-bin.zip     && curl -LO "https://sbt-downloads.cdnedge.bluemix.net/releases/v${SBT_VERSION}/sbt-${SBT_VERSION}.zip"     && unzip -q sbt-${SBT_VERSION}.zip -d /opt/     && chmod +x /opt/sbt/bin/sbt     && rm sbt-${SBT_VERSION}.zip     && curl -LO "https://github.com/koalaman/shellcheck/releases/download/v${SHELLCHECK_VERSION}/shellcheck-v${SHELLCHECK_VERSION}.linux.x86_64.tar.xz"     && tar -C /tmp/ -xvf shellcheck-v${SHELLCHECK_VERSION}.linux.x86_64.tar.xz     && cp /tmp/shellcheck-v${SHELLCHECK_VERSION}/shellcheck /usr/local/bin/shiftleft/shellcheck     && chmod +x /usr/local/bin/shiftleft/shellcheck     && curl -LO "https://github.com/dominikh/go-tools/releases/download/${SC_VERSION}/staticcheck_linux_amd64.tar.gz"     && tar -C /tmp -xvf staticcheck_linux_amd64.tar.gz     && chmod +x /tmp/staticcheck/staticcheck     && cp /tmp/staticcheck/staticcheck /usr/local/bin/shiftleft/staticcheck     && rm staticcheck_linux_amd64.tar.gz' returned a non-zero code: 6
##[error]The process '/usr/bin/docker' failed with exit code 6
```

--- a/ci/Dockerfile-java
+++ b/ci/Dockerfile-java
@@ -12,7 +12,7 @@ ENV GOSEC_VERSION=2.7.0 \
     GITLEAKS_VERSION=7.3.0 \
     GRADLE_VERSION=6.8.3 \
     GRADLE_HOME=/opt/gradle-${GRADLE_VERSION} \
-    SBT_VERSION=1.3.4 \
+    SBT_VERSION=1.5.4 \
     MAVEN_VERSION=3.6.3 \
     MAVEN_HOME=/opt/apache-maven-${MAVEN_VERSION} \
     SC_VERSION=2020.2.3 \
@@ -41,7 +41,7 @@ RUN curl -LO "https://services.gradle.org/distributions/gradle-${GRADLE_VERSION}
     && unzip -q apache-maven-${MAVEN_VERSION}-bin.zip -d /opt/ \
     && chmod +x /opt/apache-maven-${MAVEN_VERSION}/bin/mvn \
     && rm apache-maven-${MAVEN_VERSION}-bin.zip \
-    && curl -LO "https://sbt-downloads.cdnedge.bluemix.net/releases/v${SBT_VERSION}/sbt-${SBT_VERSION}.zip" \
+    && curl -LO "https://github.com/sbt/sbt/releases/download/v${SBT_VERSION}/sbt-${SBT_VERSION}.zip" \
     && unzip -q sbt-${SBT_VERSION}.zip -d /opt/ \
     && chmod +x /opt/sbt/bin/sbt \
     && rm sbt-${SBT_VERSION}.zip \

